### PR TITLE
Increase gas price

### PR DESCRIPTION
**Pull request by martinvol**

### CODE REVIEW

File: CGPs/cgp-0145.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+cgp: 154
```
Comment (work-0xj4an) [inline CGPs/cgp-0145.md:2]: I think you have a mistake here, maybe is 145?

File: CGPs/cgp-0145.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+title: Increase minimum gas threshold to 25000000000 gwei
+date-created: 2024-08-6
+author: Martín Volpe (@martinvol)
+status: DRAFT
+discussions-to:
```
Comment (work-0xj4an) [inline CGPs/cgp-0145.md:7]: We need a link where to point the discussions.

File: CGPs/cgp-0145.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+cgp: 154
+title: Increase minimum gas threshold to 25000000000 gwei
+date-created: 2024-08-6
```
Comment (work-0xj4an) [inline CGPs/cgp-0145.md:4]: Need to be in format YYYY-MM-DD

**Review by work-0xj4an — CHANGES_REQUESTED** [review]

File: CGPs/cgp-0145.md (hunk 2 above)
Comment (martinvol) [reply CGPs/cgp-0145.md:7]: But I can't make the discussion before I have a link to the proposal

Comment (work-0xj4an) [reply CGPs/cgp-0145.md:7]: Normally, people post  in the forum to get feedback on the proposal and after that they submitted the PR linking the PR to the forum discussion.  But I will approve this PR.

**Review by work-0xj4an — APPROVED** [review]

**Pull request merged**